internal/privy/server.go

```diff
@@ -73,11 +115,60 @@ func (s *Server) GetRecord(cli *xrpc.Client) http.HandlerFunc {
 
 		cid := u.Query().Get("cid")
 		collection := u.Query().Get("collection")
-		did := u.Query().Get("did")
+		repo := u.Query().Get("repo")
 		rkey := u.Query().Get("rkey")
 
-		out, err := s.inner.getRecord(r.Context(), cli, cid, collection, did, rkey)
+		// Try handling both handles and dids
+		id, err := s.dir.LookupHandle(r.Context(), syntax.Handle(repo))
```

Comment:
is this how they handle `repo` internally? wondering if they already have a method to resolve either case